src/main/java/com/knu/buga1chuk/serialization/csv/WritePersonCsvDemo.java

```diff
@@ -15,6 +15,6 @@ public static void main(String[] args) {
         PersonService personService = new PersonService();
         PersonCsvService personCsvService = new PersonCsvService();
```

Comment:
Why you use the implementation instead of the Interface?

Reply:
Fix